.rubocop.yml

```diff
@@ -1,2 +1,5 @@
 inherit_from:
  - .rubocop_cocoapods.yml
+
+Metrics/LineLength:
```

Comment:
:+1: Updated the project one because I wasn't sure if the cocoapods one was shared across the different projects